csharp/src/Google.Protobuf.Test/JsonFormatterTest.cs

```diff
@@ -234,11 +350,12 @@ public void ToCamelCase(string original, string expected)
         }
 
         [Test]
-        [TestCase(null, "{ }")]
-        [TestCase("x", "{ 'fooString': 'x' }")]
-        [TestCase("", "{ 'fooString': '' }")]
-        public void Oneof(string fooStringValue, string expectedJson)
+        [TestCase(null, "{ ", "", "}")]
```

Comment:
Personally I think this test would be clearer just with two parameters: one with normal formatting, and one with field formatting. That would make it simpler to see what the actual JSON is in the normal case, at the expense of some duplication.